haiku-apps/simplebackup/simplebackup-2.0.1.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Graphical interface to run the zip command"
+DESCRIPTION="This application makes it easy to create and update backups of \
+large amounts of data.  For example, you can create a backup of your home \
+directory and run this application regularly to make sure the backup is up to \
+date.  SimpleBackup also accepts command line options, so it is easily \
+scriptable and can be used with scheduling applications."
+HOMEPAGE="https://perelandra0x309.github.io/simplebackup"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/Perelandra0x309/simplebackup/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="463df905df35a74e81127d1c28d20a1f2c94ff1b89f6939abf081ecc70e60c62"
+SOURCE_DIR="simplebackup-${portVersion}"
```

Comment:
Well, I would recommend using x86 (gcc4) hybrid. This allows to build for both:
* x86 as primary arch,
* and x86_gcc2 as secondary arch.

Whereas if you chose x86 (gcc4) **_non_** hybrid, then you will only be able to build for x86 (gcc4) as primary arch.

BTW, I have 3 bootable partitions:
* x86_gcc2 hubrid
* x86 hybrid (i.e. the one mentionned in the beginning of this message)
* x86_64/
and a "data" partition where I have the haikuports and haikuporter trees.

Then, with a few symbolic links, I managed to have separate directories for both:
/data/haikuports/packages
/data/haikuports/repository